### ISSUE TITLE
HRRR : Failed to get CRS from crs variable LambertConfor mal_Projection

### ISSUE THREAD

**Issue by captcha1** [psyplot/psyplot#38]
```python
import psyplot.project
import matplotlib.pyplot as plt

n1 = 'GFS/Global_onedeg'
# n1 = 'HRRR/CONUS_2p5km' # XXX psy_maps/plotters.py:309: UserWarning: Failed to get CRS from crs variable LambertConformal_Projection
#n1 = 'NAM/Polar_90km' # XXX psy_maps/plotters.py:309: UserWarning: Failed to get CRS from crs variable PolarStereographic_Projection
nwp0 = f'https://thredds.ucar.edu/thredds/dodsC/grib/NCEP/{n1}/Best'

map0 = psyplot.project.plot.mapplot(nwp0,
    name = 'Geopotential_height_isobaric',
    clabel='{desc}', title = n1 + ' %FT%H:%M'
)
map0.update(z=500, method='nearest') # XXX KeyError: 'Unknown formatoption keyword lev! Possible similiar frasings are levels.'
map0.update(t = '2021-10-02', method='nearest') # XXX KeyError: 'Unknown formatoption keyword time! Possible similiar frasings are title.'

plt.show()

n0 = 'u-component_of_wind_isobaric'
vectors = psyplot.project.plot.mapvector(nwp0, name=[[n0, n0.replace('u-', 'v-')]])
plt.show()
```
#### Problem description

First off : cool software !

However, I've noticed some technical difficulties : 

1. The first plot (w/ GFS) works fine, but if you uncomment the "n1 = 'HRRR/CONUS_2p5km'", it gives a warning : 

psy_maps/plotters.py:309: UserWarning: Failed to get CRS from crs variable LambertConformal_Projection

... and then kind of hangs.

2. The plot.mapvector call dies (after some time) w/ RuntimeError: NetCDF: Access failure

3. The calls to select time & level don't seem to work.  RFE : some way to put the "level" info in the title.

4. From the command line : 

psyplot https://thredds.ucar.edu/thredds/dodsC/grib/NCEP/GFS/Global_onedeg/Best -n Pressure_reduced_to_MSL_msl

... seems like it should work, but I get : AttributeError: 'NoneType' object has no attribute 'variables'

FYI, this is on Fedora 34.

#### Output of ``psyplot -aV``

<details>
psy_maps.plugin:
  requirements:
    cartopy: 0.19.0
  version: 1.4.0
psy_simple.plugin:
  version: 1.4.0
psyplot:
  requirements:
    matplotlib: 3.4.3
    numpy: 1.20.1
    pandas: 1.2.5
    python: 3.9.7 (default, Aug 30 2021, 00:00:00)  [GCC 11.2.1 20210728 (Red Hat
      11.2.1-1)]
    xarray: 0.17.0
  version: 1.4.0
</details>

**Comment by captcha1** [comment]
Thanks for your quick fix !

The "ucar hrrr" data (ei : https://thredds.ucar.edu/thredds/dodsC/grib/NCEP/HRRR/CONUS_2p5km/Best ) doesn't error out, but has the wrong plot : the latitude goes from 25.003 to 25.03 and longitude goes from 95.01 to 94.99.

**Comment by Chilipp** [comment]
> I added a fix with psyplot/psy-maps@37121cf and it seems to work.

but thinking about it, the solution there is not bulletproof. It does not take `bounds` into account and it cannot be used for unstructured grids, for instance. So it's probably better to generalize the conversion handling that I already implemented for `radians` 

https://github.com/psyplot/psy-simple/blob/ac305c052b6bc9da47465ee1f35a1c64e489aca6/psy_simple/plotters.py#L126-L143

I can take care of that tomorrow
